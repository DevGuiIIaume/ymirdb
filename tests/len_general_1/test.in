len a
set a 1 2 3
len a
set b a 4 5 6
len a
len b
push b 7 8 9
len b
push a -2 -1 0
len a
len b
list entries
bye
