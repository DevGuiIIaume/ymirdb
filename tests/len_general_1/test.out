> no such key

> ok

> 3

> ok

> 3

> 6

> ok

> 9

> ok

> 6

> 12

> b [9 8 7 a 4 5 6]
a [0 -1 -2 1 2 3]

> bye
